specs/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/streams/kafka/publish.client.topic.space/client.rpt

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+connect "zilla://streams/kafka0"
+         option zilla:window 8192
+         option zilla:transmission "duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                             .typeId(zilla:id("kafka"))
+                             .merged()
+                                 .capabilities("PRODUCE_ONLY")
+                                 .topic("mqtt-clients")
+                                 .partition(-1, -2)
+                                 .ackMode("LEADER_ONLY")
+                                 .build()
+                             .build()}
+
+connected
+
+write notify PRODUCE_CONNECTED
+write zilla:data.ext ${kafka:dataEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .deferred(0)
+                                .partition(-1, -1)
+                                .key("/clients/client-2/sensors/one")
+                                .hashKey("client-2")
+                                .header("zilla:filter", "clients")
+                                .header("zilla:filter", "client-2")
+                                .header("zilla:filter", "sensors")
+                                .header("zilla:filter", "one")
+                                .header("zilla:local", "client-1")
+                                .header("zilla:format", "TEXT")
```

Comment:
We probably don't need explicit format in these tests, as it is outside the scope of what we are testing.